features/successful_application_statuses.feature

```diff
@@ -43,7 +43,7 @@ Feature: successful application statuses
 
     Examples:
       | original status       | actor     | action                   | new status               |
-      | unsubmitted           | candidate | submit                   | application complete     |
+      | unsubmitted           | candidate | submit                   | awaiting references      |
```

Comment:
I guess it doesn't make sense to add a `references_received` action here because the outcome is conditional? I am wondering whether we should move the conditional logic out of the state machine class into the service class. We can take a view on this again perhaps when we start to add a bit more complexity later.

Reply:
I suppose I chose to go that way because I prefer fat models and skinny service classes – arguably the state machine class is part of the model. However that runs the risk of ending up with a massive god state machine class. I'm curious as to the best place to put the logic, but agree that we should perhaps revisit it when there's more stuff there.